<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hospital</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_patients">
       <property name="text">
        <string>Больные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_doctors">
       <property name="text">
        <string>Врачи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_referral">
       <property name="text">
        <string>Направления</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_saveandexit">
       <property name="text">
        <string>Сохранить и выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
